src/main/MainMenu.tsx

```diff
@@ -34,6 +34,9 @@ const MainMenu: React.FC<{}> = () => {
     flexShrink: 0,
     alignItems: 'center',
     padding: '20px',
+    height: '95vh',
+    position: 'fixed',
```

Comment:
Minor thing, but this causes the right borderline of the main menu to not line up precisely with the bottom of the page anymore.

Reply:
For me, this seems to work better than before in both Firefox and Chromium.